assets/src/adaptivity/operators/math.ts

```diff
@@ -28,6 +28,7 @@ const operators = {
   notIsExactly: notExactlyMathOperator,
   isEquivalentOf: isEquivalentOfMathOperator,
   notIsEquivalentOf: notIsEquivalentOfMathOperator,
```

Comment:
There was a type mistake. for precaution, I didn't remove 'notIsEquivalentOf' just in case if it was used any where which is very unlikely.